src/App.jsx

```diff
@@ -14,57 +16,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedList, setSelectedList] = useState('Jam');
+  const handleResetList = () => setSelectedList('');
+  const handleAddGoodToList = good => setSelectedList(good);
+  const isInList = good => selectedList.includes(good);
```

Comment:
Pay attention to formatting. Also, you can just compare it as follow:
`selectedGood === good`